ChiModules/LinearBoltzmannSolver/lbs_01b_initmaterials.cc

```diff
@@ -118,11 +118,26 @@ void LinearBoltzmann::Solver::InitMaterials(std::set<int>& material_ids)
       << material_xs[matid_to_xs_map[mat_id]]->transfer_matrices.size() << "\n";
   }//for material id
 
+  num_groups = int(groups.size());
```

Comment:
This `int(groups.size())` cast should not be necessary and its c-style. Just remove it please.

Reply:
Done.